src/Containers/SearchPage/saga.js

```diff
@@ -0,0 +1,32 @@
+import {
+  querySearchError,
+  querySearchSuccess,
+  QUERY_SEARCH
+} from './action'
+import serialize from '../../util/serialize'
+import { call, put, takeLatest } from 'redux-saga/effects'
+import axios from 'axios'
+
+function* doQuerySearch({ param = {} }) {
+  var q;
+  param.q===""||param.q===undefined||param.q===null? q="pato":q=param.q;
```

Comment:
kinda

Reply:
if someone for some reason get in the search view without looking for something the "pato" search happen.